### PR TITLE
SUIT-15212 - Add minification for all the translation builds

### PR DESCRIPTION
**Pull request by AntonSBilan**

### CODE REVIEW

File: packages/smst-to-html/tsconfig.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
     "moduleResolution": "node"
   },
   "include": [
+    "@suitest/smst",
```
Comment (andriichumak) [inline packages/smst-to-html/tsconfig.json:9]: Why is it needed? Shouldn't it work when importing the package explicitly?

File: packages/translate/tsconfig.json [hunk 2]
```diff
@@ -6,6 +6,7 @@
     "moduleResolution": "node"
   },
   "include": [
+    "@suitest/smst",
```
Comment (andriichumak) [inline packages/translate/tsconfig.json:9]: Do we need this explicit include?

Comment (AntonSBilan) [reply packages/translate/tsconfig.json]: removed

File: packages/smst-to-html/rollup.config.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import getConfig from "../../rollupUtils";
+
+const config = process.env.FORMAT === 'umd'
+	? getConfig('toHtml.ts', 'umd', 'umd', ['@suitest/smst'], {'@suitest/smst': '@suitest/smst'})
```
Comment (JeKachan) [inline packages/smst-to-html/rollup.config.js:4]: hard to read functions with many arguments, please consider change it for accepting single settings object.

**Pull request merged**